knowledge-base/basic-authentication-with-fiddler-everywhere.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Basic Authentication with Fiddler Everywhere
+description: This article briefs about the steps of basic authentication with fiddler everywhere.
+type: how-to
+slug: basic-authentication-with-fiddler-everywhere
+tags: basic authentication, fiddler everywhere, authentication with fiddler everywhere
+publish: true
+res_type: kb
+---
+
+## Environment
+
+<table>
+	<tbody>
+		<tr>
+			<td>Product Version</td>
+			<td>1.0.1</td>
+		</tr>
+		<tr>
+			<td>Product</td>
+			<td>Fiddler Everywhere</td>
+		</tr>
+	</tbody>
+</table>
+
+## Description
+
+Fiddler Everywhere certificate allows the inspection and debugging of network traffic. The __Composer__ feature in Fiddler Everywhere enables you to manually build and send HTTP or HTTPS requests. Click [here]({%slug composer%}) to know more about Fiddler Everywhere Composer.
+
+In the context of an HTTP transaction, __Basic Access Authentication__ is a method for an HTTP user agent (for example, a web browser) to provide a user name and password when making a request. In __Basic HTTP Authentication__, a request contains a header field in the form of Authorization: Basic <credentials>, where credentials are the __Base64__ encoding of `ID` and `password` joined by a single colon(`:`).
```

Comment:
I think it would be more clear to state that the header is called `Authorization` and that its value should take the following form: `Basic <base64 encoded value>` and then explain how the value is created.

Also, why  using **_ID_** where it should be the **_username_** - it is a bit confusing to mention **_username_** (one word) and then in the very same sentence switch to **_ID_**

Use some simple formatting for the **Authorization** like 
`Authorization: Basic <base64 string>`